mirdata/datasets/indexes/acousticbrainz_genre_dataset_little_test.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
@PRamoneda should I? I guess this is from your AcousticBrainz remote index implementation.

Reply:
I think @nkundiushuti comment is because this file appears to be included "for the first time" in this PR, but if it is part of AcousticBrainz implementation it should be added in that PR instead. Maybe remove it and check if tests are passing, and if so it's better if the files is not in this PR